tests/accelerators/test_ipu.py

```diff
@@ -312,7 +312,7 @@ def test_accumulate_grad_batches_dict_fails(tmpdir):
     model = IPUModel()
     trainer = Trainer(default_root_dir=tmpdir, ipus=1, accumulate_grad_batches={0: 1})
     with pytest.raises(
-        MisconfigurationException, match="IPUs currently only support accumulate_grad_batches being an integer value."
+        MisconfigurationException, match="IPUs currently only support `Trainer.accumulate_grad_batches` being an int"
```

Comment:
```suggestion
        MisconfigurationException, match="IPUs currently only support `Trainer.accumulate_grad_batches` being an int."
```
don't forget the period :)

Reply:
`pytest.raises` doesn't require the full match, matching a subset works. I prefer matching against the minimum possible message that clearly shows what's intended to test so that any future minor changes don't necessarily fail the test.

And if we wanted to test the grammar itself, it'd be `\.`, otherwise it'd match against single character as it gets interpreted as regex by default.